Update governance-json-ld.md (#230)
bcgov/digital-trust-toolkit: docs/governance/mining/bc-mines-act-permit/governance-json-ld.md

## 1.1 Introduction

[The Mines Act](https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/96293_01) regulates mining activities in British Columbia (B.C.) A permit must be in place before any work in, on, or about a mine in B.C. can occur. 
A B.C. Mines Act Permit credential published according to this Governance Framework (GF) provides evidence that a mining permit has been issued to a registered business in the province of B.C.

> Published B.C. Mines Act Permit data uses digital credential technology in the format of [Verifiable Credential](https://www.w3.org/TR/vc-data-model-2.0/) which expresses linked data that is both human and machine-readable. An important feature of publishing public data using [JSON-LD](https://json-ld.org/) is the ability to enable compliance with international standards like the [UN Transparency Protocol (UNTP)](https://uncefact.github.io/spec-untp/).

Author: ```suggestion
Published B.C. Mines Act Permit data uses digital credential technology in the format of [W3C Verifiable Credential](https://www.w3.org/TR/vc-data-model-2.0/) which expresses linked data that is both human and machine-readable. An important feature of publishing public data using this format is the ability to enable compliance with international standards like the [UN Transparency Protocol (UNTP)](https://uncefact.github.io/spec-untp/).
```

> A published B.C. Mines Act Permit credential uses verifiable credential technology in the format of [JSON-LD](https://json-ld.org/), or JavaScript Object Notation, which expresses data that is both human and machine-readable in such a way that it can be linked. An important feature of publishing public data using JSON-LD is the ability to enable compliance with international standards like the [UN Transparency Protocol (UNTP)](https://uncefact.github.io/spec-untp/).

Reviewer: ```suggestion
A published B.C. Mines Act Permit credential uses digital credential technology in the format of [Verifiable Credential](https://www.w3.org/TR/vc-data-model-2.0/) which expresses linked data that is both human and machine-readable. An important feature of publishing public data using [JSON-LD](https://json-ld.org/) is the ability to enable compliance with international standards like the [UN Transparency Protocol (UNTP)](https://uncefact.github.io/spec-untp/).
```

## 2.4. Governance Requirements

Legislation and regulations govern the disposition, administration, and management of mines in B.C.
- **[The Mines Act](https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/96293_01)**
- **[B.C. Mine Information](https://mines.nrs.gov.bc.ca/)**
- **[B.C. Mine Authorizations](https://mines.nrs.gov.bc.ca/authorizations)**

## 2.5. Business Requirements

The primary use of the B.C. Mines Act Permit credential is to provide evidence of permit issuance from the province to a registered business entity according to the [B.C. Mines Act](https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/96293_01). 

## 2.6. Technical Requirements (Credential)
The format for this credential is [JSON-LD specification](https://www.w3.org/TR/json-ld11/) and is compliant with [Verifiable Credentials Data Model v2.0](https://www.w3.org/TR/vc-data-model-2.0/#json-ld)

### 2.6.1 Context Files
- [BCMinesActPermit, version 1](https://github.com/bcgov/digital-trust-toolkit/blob/main/related_resources/contexts/BCMinesActPermit/v1.jsonld)

>   - which extends the Digital Conformity Credential of [UNTP, version 0.5.0](https://test.uncefact.org/vocabulary/untp/dcc/0.5.0/)

Reviewer: ```suggestion
  - which is intended to extend the Digital Conformity Credential of [UNTP, version 0.5.0](https://test.uncefact.org/vocabulary/untp/dcc/0.5.0/)
```

## 2.4. Governance Requirements

Legislation and regulations govern the disposition, administration, and management of mines in B.C.
- **[The Mines Act](https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/96293_01)**
- **[B.C. Mine Information](https://mines.nrs.gov.bc.ca/)**
- **[B.C. Mine Authorizations](https://mines.nrs.gov.bc.ca/authorizations)**

## 2.5. Business Requirements

The primary use of the B.C. Mines Act Permit credential is to provide evidence of permit issuance from the province to a registered business entity according to the [B.C. Mines Act](https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/96293_01). 

## 2.6. Technical Requirements (Credential)

> The format for this credential is [JSON-LD specification](https://www.w3.org/TR/json-ld11/) and is compliant with [Verifiable Credentials Data Model v2.0](https://www.w3.org/TR/vc-data-model-2.0/#json-ld)

Author: ```suggestion
The format for this credential is [JSON-LD specification](https://www.w3.org/TR/json-ld11/) and is compliant with [W3C Verifiable Credentials Data Model v2.0](https://www.w3.org/TR/vc-data-model-2.0/#json-ld)
```
